src/GrammarDecl.js

```diff
@@ -21,6 +22,10 @@ function GrammarDecl(name) {
 
 // Helpers
 
+GrammarDecl.prototype.at = function(startIdx, endIdx) {
```

Comment:
IMO this is method name makes call sites confusing, e.g.,

```.define("NameNoFormals", [], this.app("name").withInterval(decl.at(49, 53)))```

It might help to rename it to `sourceInterval`:

```.define("NameNoFormals", [], this.app("name").withInterval(decl.sourceInterval(49, 53)))```